Model/Processor/ImportProcessor.php

```diff
@@ -85,8 +85,18 @@ public function __construct(
     public function process()
     {
         $files = $this->finder->find();
-        if (0 === count($files)) {
-            throw new \InvalidArgumentException('No files found for format: *.' . $this->getFormat());
+
+        try {
+            if (0 === count($files)) {
+                throw new \InvalidArgumentException('No files found for format: *.' . $this->getFormat());
+            }
+        } catch (\InvalidArgumentException $exception) {
+            $inputOptions = $this->getInput()->getOptions();
+            if (true === $this->getInput()->getOption('allow-empty-directories')) {
+                $this->getOutput()->writeln('No files found for format: *.' . $this->getFormat());
+            } else {
+                throw new \InvalidArgumentException($exception->getMessage());
```

Comment:
You could just re-throw the caught exception

Reply:
Good point, changed that now.